packages/core/src/components/loading-skeleton/_loading-skeleton.scss

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-loading-skeleton-bones {
```

Comment:
I would say so. I like the skeleton/bone metaphor but IMO it's pushed a little too far with "bones"